chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -0,0 +1,644 @@
+package bitcoindnotify
```

Comment:
I haven't done anything with the build flags at this point; I believe you said in chat that we can avoid it at this point. It may be easier to move to `gomq` and add `PUB/SUB` socket support to it than do the refactoring necessary to be able to turn this off with build flags.

Reply:
No build flags required any more!